backend/Dockerfile

```diff
@@ -4,18 +4,22 @@ EXPOSE 80
 EXPOSE 443
 
 FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
-WORKDIR /src
+
+COPY . .
+# WORKDIR /src
 # Copy the main source project files
-COPY backend/*/*.csproj ./
+# COPY */*.csproj ./
+#
 # move them into the proper sub folders, based on the name of the project
-RUN for file in $(ls *.csproj); do mkdir -p backend/${file%.*}/ && mv $file backend/${file%.*}/; done
-RUN dotnet restore "backend/LexBoxApi/LexBoxApi.csproj"
-COPY . .
-WORKDIR "/src/backend/LexBoxApi"
-RUN dotnet build "LexBoxApi.csproj" -c Release -o /app/build
+# RUN for file in $(ls *.csproj); do mkdir -p ${file%.*}/ && mv $file ${file%.*}/; done
+RUN dotnet restore "LexBoxApi/LexBoxApi.csproj"
```

Comment:
since we copy in all file at line 8 there's no long a benefit of splitting the restore and build action into 2 commands. You can remove this line too, otherwise we should revert back if we want build caching to work again.

Reply:
oh ok, I noticed those files got updated so I thought it was still important to run that restore... maybe we'll have a chance to look at this and finalize the way that works best.